docs/monitoring/rsv.md

```diff
@@ -0,0 +1,334 @@
+
+#### Configuring the RSV web server to use HTTPS instead of HTTP
+
+If you would like your local RSV web server to use HTTPS instead of the default HTTP (for compatibility or security reasons), complete the steps below. This procedure assumes that you already have an HTTP service certificate (or a copy of the host certificate) in `/etc/grid-security/http/`. If not, omit the `SSLCertificate*` modifications below, and your web server will start with its own, self-signed certificate.
+
+1. Install `mod_ssl`:
+
+        :::console
+        [root@client ~] # yum install mod_ssl
+
+2. Make an alternate set of HTTP service certificate files:
+
+        :::console
+        [root@client ~] # cp -p /etc/grid-security/http/httpcert.pem /etc/grid-security/http/httpcert2.pem
+        [root@client ~] # cp -p /etc/grid-security/http/httpkey.pem /etc/grid-security/http/httpkey2.pem
+        [root@client ~] # chown apache:apache /etc/grid-security/http/http*2.pem
+
+3. Back up existing Apache configuration files:
+
+        :::console
+        [root@client ~] # cp -p /etc/httpd/conf/httpd.conf /etc/httpd/conf/httpd.conf.orig
+        [root@client ~] # cp -p /etc/httpd/conf.d/ssl.conf /etc/httpd/conf.d/ssl.conf.orig
+
+4. Change the default port for HTTP connections to 8000 by editing `/etc/httpd/conf/httpd.conf`
+
+        :::file
+        Listen 8000
+        
+
+5. Set up HTTPS access by editing `/etc/httpd/conf.d/ssl.conf`:
+
+        :::file
+        Listen 8443
+        <VirtualHost _default_:8443>
+        SSLCertificateFile /etc/grid-security/http/httpcert2.pem	
+        SSLCertificateKeyFile /etc/grid-security/http/httpkey2.pem
+
+    After these changes, when you start the Apache service, it will listening on ports `8000` (for HTTP) and `8443` (for HTTPS), rather than the default port `80` (for HTTP only).
+
+    !!! warning
+        if you make the changes above, you must restart the Apache server after each CA certificate update to pick up the changes.
+
+
+Using RSV
+---------
+
+### Managing RSV and associated services
+
+In addition to the RSV service itself, there are a number of supporting services in your installation. The specific services are:
+
+| Software      | Service name                                   | Notes                   |
+|:--------------|:-----------------------------------------------|------------------------ |
+| Fetch CRL     | On EL 6: `fetch-crl-boot` and `fetch-crl-cron` | See (CA documentation)[]|
```

Comment:
Remove the words 'On EL 6:' since the instructions apply to both el6 and el7; also change link to point to the twiki since the InstallCertAuth hasn't been migrated yet